packages/components/src/components/Select/Select.stories.tsx

```diff
@@ -0,0 +1,253 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconIcon } from '@frontify/fondue-icons';
+import { type Meta, type StoryObj } from '@storybook/react';
+import { useState } from 'react';
+
+import { Button } from '../Button/Button';
+
+import { SelectCombobox } from './Combobox';
+import { Select, SelectInput } from './Select';
+import { SelectItem, SelectItemGroup } from './SelectItem';
+import { SelectSlot } from './SelectSlot';
+
+type Story = StoryObj<typeof meta>;
+const meta: Meta<typeof SelectInput> = {
+    component: SelectInput,
+    subcomponents: {
+        // @ts-expect-error Storybook types are incorrect
+        'Select.Combobox': SelectCombobox,
+        // @ts-expect-error Storybook types are incorrect
+        'Select.Slot': SelectSlot,
+        // @ts-expect-error Storybook types are incorrect
+        'Select.Group': SelectItemGroup,
+        // @ts-expect-error Storybook types are incorrect
+        'Select.Item': SelectItem,
+    },
+    tags: ['autodocs'],
+    parameters: {
+        status: {
+            type: 'in_progress',
+        },
+    },
+    args: {
+        'aria-label': 'Select an item',
+    },
+};
+export default meta;
+
+export const SimpleSelect: Story = {
+    name: 'Select',
```

Comment:
You can remove the `name:` and use the `export const <Name>` instead (eslint storybook warns about it)

Reply:
i did this because it generates a conflict if Both the Story and Component are called `Select`